recipes/moc/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from os import rename
+from os.path import isdir, join
+from shutil import move
+from glob import glob
+
+class MocConan(ConanFile):
+    name = "moc"
+    homepage = "https://www.github.com/zuut/moc"
+    description = "Moc, the marked-up object compiler"
+    topics = ("conan", "moc", "idl", "code generation")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"    
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = { "fPIC": [True, False] }
+    default_options = { "fPIC": True }
+    _cmake = None
+    _supported_compilers = [
+        ("Linux", "gcc", "6"),
+        ("Linux", "clang", "6")
+    ]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if not self._check_compiler():
+            raise ConanInvalidConfiguration("%s package is not compatible with os %s and compiler %s version %s." % (self.name, self.settings.os, self.settings.compiler, self.settings.compiler.version))
+
+    def _check_compiler(self):
+        os = self.settings.os
+        compiler = self.settings.compiler
+        compiler_version = tools.Version(compiler.version)
+        return any(os == sc[0] and compiler == sc[1] and compiler_version >= sc[2] for sc in self._supported_compilers)
+
+    def requirements(self):
+        self.requires("flex/2.6.4")
+        self.requires("bison/3.7.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.remove_files_by_mask(join(self.package_folder, "lib"), "*-config.cmake")
+        self.copy(pattern="LICENSE", dst="licenses",
+                  src=self._source_subfolder,
+                  keep_path=False)
+        cmakeModulesDir=join(self.package_folder, "cmake-modules")
+        if isdir(cmakeModulesDir):
+            move(cmakeModulesDir,join(self.package_folder, "lib/cmake"))
+        return
+
+    def package_info(self):
+        self.env_info.MOC_TOOL = join(self.package_folder, "bin", "moc").replace("\\", "/")
+
+        self.env_info.PATH.append(join(self.package_folder, "bin"))
```

Comment:
Usually we print this for the consumer to see

Reply:
Would you clarify what you're referring to? I haven't seen any ouput statements in any of the package_info methods of recipes that I've looked at so far.